packages/desktop/src/renderer/components/ui/ErrorModal/ErrorModal.test.tsx

```diff
@@ -121,7 +121,7 @@ describe('ErrorModal', () => {
                       <p
                         class="MuiTypography-root makeStyles-info-5 MuiTypography-body2"
                       >
-                        You can send us this error traceback to help us improve. Before sending make sure it doesn't contain any private data.
+                        This error traceback was sent to centralized server.
```

Comment:
Is this sentence always true? The modal will be shown on every error but we won't be sending data to sentry every time, right? We won't be, right? [Anakin & Padme meme here]